products/gstreamer.md

```diff
@@ -0,0 +1,100 @@
+---
+title: GStreamer
+category: framework
+iconSlug: gstreamer
+permalink: /gstreamer
+versionCommand: gst-inspect-1.0 --version
+releasePolicyLink: https://gstreamer.freedesktop.org/
+changelogTemplate: https://gstreamer.freedesktop.org/releases/__RELEASE_CYCLE__/
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Supported
+
+auto:
+-   git: https://gitlab.freedesktop.org/gstreamer/gstreamer.git
+    regex: '^(?<major>[1-9]\d*)\.(?<minor>([1-9]\d*)?[02468])\.?(?<patch>0|[1-9]\d*)?$'
+
+releases:
+-   releaseCycle: "1.22"
+    releaseDate: 2023-01-23
+    eol: false
+    latest: "1.22.2"
+    latestReleaseDate: 2023-04-11
+
+-   releaseCycle: "1.20"
+    releaseDate: 2022-02-03
+    eol: true
+    latest: "1.20.6"
+
+-   releaseCycle: "1.18"
+    releaseDate: 2020-09-08
+    eol: true
+    latest: "1.18.6"
+
+-   releaseCycle: "1.16"
+    releaseDate: 2019-04-19
+    eol: true
+    latest: "1.16.3"
+
+-   releaseCycle: "1.14"
+    releaseDate: 2018-03-19
+    eol: true
+    latest: "1.14.5"
+
+-   releaseCycle: "1.12"
+    releaseDate: 2017-05-04
+    eol: true
+    latest: "1.12.5"
+
+-   releaseCycle: "1.10"
+    releaseDate: 2016-11-01
+    eol: true
+    latest: "1.10.5"
+
+-   releaseCycle: "1.8"
+    releaseDate: 2016-03-24
+    eol: true
+    latest: "1.8.3"
+
+-   releaseCycle: "1.6"
+    releaseDate: 2015-09-25
+    eol: true
+    latest: "1.6.4"
+
+-   releaseCycle: "1.4"
+    releaseDate: 2014-07-19
+    eol: true
+    latest: "1.4.5"
+
+-   releaseCycle: "1.2"
+    releaseDate: 2013-09-24
+    eol: true
+    latest: "1.2.4"
+
+-   releaseCycle: "1.0"
+    releaseDate: 2012-09-24
+    eol: true
+    latest: "1.0.10"
+
+---
+
+> [GStreamer](https://gstreamer.freedesktop.org/) is a pipeline-based
+> multimedia framework that links together a wide variety of media
+> processing systems to complete complex workflows. For instance,
+> GStreamer can be used to build a system that reads files in one
+> format, processes them, and exports them in another. The formats and
+> processes can be changed in a plug and play fashion.
+
+GStreamer follows [Semantic Versioning](https://semver.org/). The
+support and EOL policy is not clearly defined but, looking at the
+releases history, there is at least a `x.y` release every year which
+is supported for at least a year with important bug fixes.
+
+The versioning scheme that is used in general is that 1.x.y is API and
+ABI backwards compatible with previous 1.x.y releases. If x is an even
+number it is a stable release series and all releases in this series
```

Comment:
This is implied by the fact that GStreamer follows semantic versioning. Maybe we could remove this paragraph and amend the first sentence of the first paragraph just to reflect that odd release are development release ? Something like :

```markdown
GStreamer follows [Semantic Versioning](https://semver.org/). Moreover GStreamer is using the following version policy:

- Stable releases (ready for production use) are even numbers, such as 1.0, 1.2 or 1.4.
- Development releases are odd numbers, such as 1.1, 1.3 or 1.5.

The support and EOL policy is not clearly defined. But, looking at the releases history, there is at least a `x.y` release every year which is supported for at least a year with important bug fixes.
```

Reply:
Thanks for the suggestion.